package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "use-constant",
-  "version": "1.1.0",
+  "version": "1.1.1",
```

This is always being updated separately through `npm version` call (or similar). The advantage of using a command like this is that we gonna have it in a separate commit that will automatically get git tagged.
```suggestion
  "version": "1.1.0",
```